docs/configure

```diff
@@ -654,7 +657,8 @@ virtual_domains:
   domains = +local_domains
   address_data = ${lookup mysql{\
         select smtp, users.sa_tag*10 AS sa_tag, users.on_spamassassin AS on_spamassassin, \
-        users.uid AS uid, users.gid AS gid, quota from users,domains \
+        users.uid AS uid, users.gid AS gid, quota, domains.spamassassin AS dom_spamassassin \
+        from users,domains \
```

Comment:
nice idea. Shouldn't we use `users.on_spamassassin AND domains.spamassassin as on_spamassassin`?